### ISSUE TITLE
Research: Improve Linked Text for Better Readability (Footer Request 76151)

### ISSUE THREAD

**Issue by jknipes** [department-of-veterans-affairs/va.gov-team#76566]
## Research 
Look into common and best practices for readability of linked text for possible improvements

## Context from Footer Request
From ticket [76151](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/76151):

- Links on the Platform website are dark blue with no default text decoration, while body text is black. It can be hard to quickly differentiate link text from regular body text. 

- This presents obstacles for users scanning for links on the page, and could also make usability more difficult for users with difficulty differentiating colors. 
- It would be a usability and accessibility improvement to convey that text is a link beyond the use of color - **ideally by underlining links**. 

- This should be considered for all links on the site, but especially links that are not part of navigation components (i.e. inline links in page content).
- It has also been suggested by this user, and others, that `va.gov` should appear as plain text, and not as a hyperlink.

---
## Tasks
- [x] Determine accessibility best practices for links (colored text vs. underlining) _see link and screenshot below_ 
- [x] Determine among leadership whether "va.gov" should be changed to **not** automatically be converted to hyperlink

## Acceptance Criteria
- [x] We will have determined accessibility best practices for links appearance
- [x] Create separate tickets for any further research, discovery or editing.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

## Web Content Accessibility Guidelines:
https://www.w3.org/TR/WCAG20-TECHS/G183.html
## Screen shot of guidance:
<img width="911" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeitEQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--14977f40838a7c36460fa3e607e903da69210317/Screenshot%202024-06-21%20at%203.07.19%E2%80%AFPM.png" alt="Screenshot 2024-06-21 at 3.07.19 PM.png" />

Attaching a screenshot of how inline text links are currently rendering (this is on the [Modernized VA.gov Accessibility Strategy](https://depo-platform-documentation.scrollhelp.site/getting-started/modernized-va-gov-accessibility-strategy) page):

![Screenshot 2024-02-13 at 4 38 16 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/755248/1ffc28fe-3a72-4095-a170-bffd5eb78fb0)

**Comment by jknipes** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Content-Team-62bde00be46b400017674aef/issues/department-of-veterans-affairs/va.gov-team/76566?planning-poker) with Zenhub @BSmolensky @laucon

**Comment by jknipes** [comment]
Searching [Web Content Accessibility Guidelines](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/Overview.html#contents) for more guidance

**Comment by jknipes** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Content-Team-62bde00be46b400017674aef/issues/department-of-veterans-affairs/va.gov-team/76566?planning-poker) with Zenhub @taylojill

**Comment by jknipes** [comment]
Per discussion with Andrea Hewitt, Content Team needs to 

1. come up with rules regarding when to allow a hyperlink for va.gov and when to disallow
2. find out whether k15t has an option for turning off automatic hyperlinking

**Comment by jknipes** [comment]
Acceptance Criteria met, closing ticket as _done_
